packages/docusaurus/src/webpack/plugins/CleanWebpackPlugin.ts

```diff
@@ -211,31 +194,14 @@ class CleanWebpackPlugin {
 
   removeFiles(patterns: string[]): void {
     try {
-      const deleted = delSync(patterns, {
+      delSync(patterns, {
         force: false,
         // Change context to build directory
         cwd: this.outputPath,
         dryRun: false,
         dot: true,
         ignore: this.protectWebpackAssets ? this.currentAssets : [],
       });
-
-      /**
-       * Log if verbose is enabled
-       */
-      if (this.verbose) {
-        deleted.forEach((file) => {
-          const filename = path.relative(process.cwd(), file);
-
-          /**
-           * Use console.warn over .log
-           * https://github.com/webpack/webpack/issues/1904
-           * https://github.com/johnagan/clean-webpack-plugin/issues/11
-           */
-          // eslint-disable-next-line no-console
-          console.warn(`clean-webpack-plugin: removed ${filename}`);
-        });
-      }
     } catch (error) {
       const needsForce = /Cannot delete files\/folders outside the current working directory\./.test(
         error.message,
```

Comment:
webpack v5 have `output.clean`, maybe you don't need this plugin anymore

Reply:
True, I'm not sure about it though seeing that this plugin appears much more basic then webpack's, and it also appears to have functionality for removing "stale assets" so I'm not sure.